src/main/java/uk/gov/hmcts/reform/ccd/client/model/CaseDetails.java

```diff
@@ -13,19 +14,25 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CaseDetails {
 
+    @JsonProperty("id")
```

Comment:
why is this added?

Reply:
If we are adding properties that are not the names to be consistent we should add them all.

Reply:
it makes it harder to tell whats changed though if you are adding everything, adding only the things that you have a reason for makes the code easier to read and easier to tell what is being remapped

Reply:
when it comes to using lombok it can copy the annotations across to the constructor so you can use @Value  but I dont think it likes the @Alias annotations.